invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities-carousel/CommunitiesCarousel.js

```diff
@@ -97,72 +98,90 @@ export default class CommunitiesCarousel extends Component {
         )
     );
 
-    return data.hits.length ? (
-      <Container fluid className="carousel rel-pt-2 rel-pb-2 ml-0-mobile mr-0-mobile">
-        {title && (
-          <Container className="rel-mb-1">
-            <Header as="h2">{title}</Header>
+    return (
```

Comment:
can you please fix the linter error above?
can you also have a look to the warnings below?